### PR TITLE
Report brotli improvement.

### PR DESCRIPTION
**Pull request by rpiaggio**
The brotli improvement is not so impressive `1259.69 KiB --> 1030.15 KiB (81.78%)` but still...

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
 const path = require("path")
 const fs = require("fs")
 const process = require("process")
-const size = require('human-format');
+const size = require("human-format")
```
Comment (toddburnside) [inline build.js:4]: Is this `require` needed? It seems like the one three lines below is what is being used.

**Review by toddburnside — APPROVED** [review]
LGTM

**Review by cquiroz — APPROVED** [review]
Looks good, should build.js be done using ES imports?

File: build.js (hunk 1 above)
Comment (rpiaggio) [reply build.js:4]: You're right, I brought it in twice. Thanks!

**Comment by rpiaggio** [comment]
> Looks good, should build.js be done using ES imports?

If I try using `import` I get `SyntaxError: Cannot use import statement outside a module`. I admit I'm a bit new to this. Is there something I'm missing?

**Comment by cquiroz** [comment]
Ah don't worry, just use `require`

**Pull request merged**